docs/README.Linux.md

```diff
@@ -70,7 +70,7 @@ git clone https://github.com/xbmc/xbmc kodi
+* debhelper (>= 8.0.0~), autoconf, automake, autopoint, gettext, autotools-dev, cmake, curl, default-jre | openjdk-6-jre | openjdk-7-jre, gawk, gcc (>= 4.9) | gcc-4.9, g++ (>= 4.9) | g++-4.9, cpp (>= 4.9) | cpp-4.9, gdc, gperf, libasound2-dev | libasound-dev, libass-dev (>= 0.9.8), libavahi-client-dev, libavahi-common-dev, libbluetooth-dev, libbluray-dev, libbz2-dev, libcdio-dev, libcec4-dev | libcec-dev, libp8-platform-dev, libcrossguid-dev, libcurl4-openssl-dev | libcurl4-gnutls-dev | libcurl-dev, libcwiid-dev, libdrm-dev, libdbus-1-dev, libegl1-mesa-dev, libenca-dev, libflac-dev, libfontconfig-dev, libfmt3-dev | libfmt-dev, libfreetype6-dev, libfribidi-dev, libfstrcmp-dev, libgbm-dev, libgcrypt-dev, libgif-dev (>= 5.0.5), libgl1-mesa-dev | libgl-dev, libgles2-mesa-dev, libglew-dev, libglu1-mesa-dev | libglu-dev, libgnutls-dev | libgnutls28-dev, libgpg-error-dev, libinput-dev, libiso9660-dev, libjpeg-dev, liblcms2-dev, liblirc-dev | liblircclient-dev, libltdl-dev, liblzo2-dev, libmicrohttpd-dev, libmysqlclient-dev, libnfs-dev, libogg-dev, libomxil-bellagio-dev [armel], libpcre3-dev, libplist-dev, libpng12-dev | libpng-dev, libpulse-dev, libshairplay-dev, libsmbclient-dev, libsqlite3-dev, libssh-dev, libssl-dev, libtag1-dev (>= 1.8) | libtag1x8, libtiff5-dev | libtiff-dev | libtiff4-dev, libtinyxml-dev, libtool, libudev-dev, libva-dev, libvdpau-dev, libvorbis-dev, libxkbcommon-dev, libxmu-dev, libxrandr-dev, libxslt1-dev | libxslt-dev, libxt-dev, waylandpp-dev, wayland-protocols, lsb-release, nasm [!amd64], python-dev, python-pil | python-imaging, python-support | python-minimal, rapidjson-dev, swig, unzip, uuid-dev, yasm, zip, zlib1g-dev
```

Comment:
- debhelper isn't needed
- libssh support was moved to a binary addon
- IMO the dependencies for the different windowing systems (X11, Wayland and GBM) should in separate lists
- maybe also distinguish between required and optional dependencies

Reply:
Agreed. However, I won't have time to do that - and test - in the next couple of days and starting Wednesday I'm off the grid until Sept 3. With very few exceptions, my wife really means off the grid.

If someone beats me to it, please close this.

Reply:
Would be great if you could just fix the wrong waylandpp-dev and wayland-protocols alternatives and leave everything else unchanged.

Reply:
:+1:
I'll find the time tonight or tomorrow.